### PR TITLE
Encoders 101

### PR DESCRIPTION
**Pull request by Eeshwar-Krishnan**
Thank you to 7236 Ethan for the idea

### CODE REVIEW

File: source/docs/software/tutorials/encoders.rst [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+                  waitForStart();
+
+                  while (opModeIsActive()) {
+                        //Get the current position of the motor
+                        double CPR = [Your Counts Per Revolution Here];
+
+                        double diameter = 1.0; //Replace with your object's diameter
+                        double circumference = Math.PI * diameter;
+
+                        double position = motor.getCurrentPosition();
```
Comment (abidingabi) [inline source/docs/software/tutorials/encoders.rst:302]:
```suggestion
                        int position = motor.getCurrentPosition();
```

File: source/docs/software/tutorials/encoders.rst [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+                        double revolutions = position/CPR;
+
+                        double angle = revolutions * 360;
+                        double angle_normalized = angle % 360;
```
Comment (abidingabi) [inline source/docs/software/tutorials/encoders.rst:306]:
```suggestion
                        double angleNormalized = angle % 360;
```

File: source/docs/software/tutorials/encoders.rst [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+
+                        double distance = revolutions * circumference;
+
+                        //Show the position of the motor on telemetry
+                        telemetry.addData("Encoder Position", position);
+                        telemetry.addData("Encoder Revolutions", revolutions);
+                        telemetry.addData("Encoder Angle (Degrees)", angle);
+                        telemetry.addData("Encoder Angle - Normalized (Degrees)", angle_normalized);
```
Comment (abidingabi) [inline source/docs/software/tutorials/encoders.rst:314]:
```suggestion
                        telemetry.addData("Encoder Angle - Normalized (Degrees)", angleNormalized);
```

**Review by abidingabi — CHANGES_REQUESTED** [review]
A bunch of little suggestions. The page also needs blocks examples code.

**Review by colloquist — APPROVED** [review]

**Review by abidingabi — APPROVED** [review]

**Pull request merged**